system/Entity/Entity.php

```diff
@@ -530,6 +534,37 @@ public function __isset(string $key): bool
      */
     public function __unset(string $key): void
     {
+        if ($this->isMappedDbColumn($key)) {
+            return;
+        }
+
+        $key = $this->mapProperty($key);
+
         unset($this->attributes[$key]);
     }
+
+    /**
+     * Whether this key is mapped db column name?
+     */
+    protected function isMappedDbColumn(string $key): bool
+    {
+        $maybeColumnName = $this->mapProperty($key);
+
+        // Property name which has mapped column name
+        if (($key !== $maybeColumnName)) {
```

Comment:
multiple parentheses can be removed:

```suggestion
        if ($key !== $maybeColumnName) {
```

Reply:
Thanks. 
But this should be removed by cs-fixer or rector.